wordpress-mobile/WordPress-iOS
# Fix Create Button jumping from left to right

Related to #13320

To test:
- Log in to a fresh install of the app on an iPhone
- Check that the Create Button is positioned on the right hand side of the device

PR submission checklist:
